google_scraper.py

```diff
@@ -29,24 +29,24 @@ def search(url):
     browser = webdriver.Chrome()
     browser.set_window_size(1024, 768)
     print("\n===============================================\n")
-    print("[%] Successfully launched Chrome Browser")
+    print("==> Chrome Driver Launched")
```

Comment:
Let's keep the original `[%]` and make `Chrome Driver` -> `Chromedriver`

Reply:
ok, addressing all these fixes now.